### ISSUE TITLE
Plugin check UX improvement for results

### ISSUE THREAD

**Issue by jakaria-istauk** [10up/plugin-check#325]
We can add an expand and collapse button for displaying results. by default, all tables are expanded. There will be a common button that collapses all tables. and after every file name, there is also a button that separately opens and closes every table. 

![image](https://github.com/WordPress/plugin-check/assets/18147494/83eef53d-af5e-48df-a183-0d7f64a38c62)

**Comment by davidperezgar** [comment]
I don't think that is necessary and it gives more elements to the report. I don't think is practical for the most of users.
